jujugui/static/gui/src/app/app.js

```diff
@@ -1194,6 +1194,15 @@ YUI.add('juju-gui', function(Y) {
         document.getElementById('header-search-container'));
     },
 
+
+    _renderHeaderHelp: function() {
+      ReactDOM.render(
+        <window.juju.components.HeaderHelp
+          changeState={this.state.changeState.bind(this.state)}
```

Comment:
changeState does not seem to be required by the component, and it would also be redundant given you are passing the state itself.